CMakeLists.txt

```diff
@@ -1,14 +1,9 @@
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Do you use any new features from this version? Otherwise, upgrading this version may cause user building failure.

Reply:
I will use `SOURCE_SUBDIR` in `arrow_ep` building.

em... I see it is not good enough, but if you need to keep lower version, I can use another solution which is worse to understand.